src/instructlab/schema/v3/knowledge.json

```diff
@@ -0,0 +1,134 @@
+{
+    "title": "Knowledge",
+    "description": "A knowledge skill.",
+    "type": "object",
+    "$ref": "./version.json",
+    "required": [
+        "created_by",
+        "domain",
+        "task_description",
+        "seed_examples",
+        "document",
+        "document_outline"
+    ],
+    "unevaluatedProperties": false,
+    "properties": {
+        "created_by": {
+            "description": "The GitHub username of the contributor.",
+            "type": "string",
+            "minLength": 1
+        },
+        "domain": {
+            "description": "The knowledge domain which is used in prompts to the teacher model during synthetic data generation. The domain should be brief such as the title to a textbook chapter or section.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+                "Chemistry",
+                "History",
+                "Pop culture"
+            ]
+        },
+        "task_description": {
+            "description": "A description of the task which is used in prompts to the teacher model during synthetic data generation. The description should be detailed and prescriptive to improve the teacher model's responses.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+                "To teach a language model about softball history",
+                "To teach a language model about tabby cats"
+            ]
+        },
```

Comment:
This is no longer present in the example I'm working from - probably needs to be removed, but I'm confirming first

Reply:
removed